.ado/templates/react-native-init.yml

```diff
@@ -65,6 +65,9 @@ steps:
       npx --no-install beachball publish --branch origin/$(System.PullRequest.TargetBranch) --no-push --registry http://localhost:4873 --yes --access public --changehint "Run `yarn change` from root of repo to generate a change file."
     displayName: Publish packages to verdaccio
 
+  - script: git pull
```

Comment:
Do we need this one?  The beachball publish --no-push command surely ends up in a state where the vnext package versions are in the correct state?

Otherwise you have a synchronization issue no?

Reply:
Missed the no-push. Will remove this one.